DragAndDrop.py

```diff
@@ -0,0 +1,20 @@
+import kivy
+import json
+import random
+from Question import Question
+
+class DragAndDrop(Question):
+    def __init__(self, ordered_image_ids, current_answer):
+        self.question_text = Question.question_text
+        self.question_id = Question.question_id
+        self.question_audio = Question.question_audio
```

Comment:
This isn't the right syntax. The Question class should have a constructor that takes in question_text, question_id, question_audio. Then you need to use the super method to pass those fields to the parent class

Reply:
something like Question.__init__(self, question_id, question_text, question_audio, explanation_text, explanation_audio)